### ISSUE TITLE
Invalid file prop (as data url) because adding `filename=generated.pdf;`

### ISSUE THREAD

**Issue by itamardavidyan** [wojtekmaj/react-pdf#784]
**Description**

When I pass data-url value to file prop like this:
```
cosnt fileContent = 'data:application/pdf;filename=generated.pdf;base64,JVBERi0xLjMKJbrf...'

<Document file={fileContent} >
   ...
</Document>
```
I get an error: `Failed to load PDF file.`
But when I delete `filename=generated.pdf;` from `fileContent` the problem solved.

**Expected behavior**
`Document` component will ignore `filename`

**Environment**

 - Browser: Chrome 90
 - React-PDF version: 5.3.0
 - React version: 17.0.2

**Comment by paddy57** [comment]
Hello, are you able to solve this ?

**Comment by itamardavidyan** [comment]
> Hello, are you able to solve this ?

Yes, I manipulate the fileContent using `fileContent.replace('filename=generated.pdf;','')`
But this is only works if you know the filename.

If you don't know the file name you can use regex in the replace function - **But BE AWARE!! using regex on long string (like file content) can make performance issues**
You can try to optimize it by cutting the head of the file content (`data:application/pdf;filename=generated.pdf;base64,`) then use the replace with regex and concat to the blob.

Hope it's help you.

**Comment by wojtekmaj** [comment]
Fixed in v5.3.1.
